packages/core/src/styles/components/_legend.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../node_modules/@carbon/themes/scss/themes";
+
+g.cc-legend {
+	user-select: none;
+
+	g.legend-item {
+		rect.checkbox {
+			stroke-width: 1px;
+			stroke-opacity: 1;
+
+			&:not(.active) ~ g.check {
+				display: none;
+			}
+		}
+	}
+
+	&.clickable {
+		g.legend-item:hover {
+			cursor: pointer;
+		}
+	}
+}
+
+
+/////////////////////////////////////////////
+///          Theme styles                ////
+/////////////////////////////////////////////
+
+.carbon--theme--white {
+	g.cc-legend {
+		g.legend-item {
+			rect.checkbox {
+				stroke: $ui-background;
+
+				&:not(.active) {
+					stroke: $text-02;
+					fill: $ui-background;
+				}
+			}
+		}
+	}
+}
+
+.carbon--theme--g10 {
+	@include carbon--theme($carbon--theme--g10) {
+		g.cc-legend {
+			g.legend-item {
+				rect.checkbox {
+					stroke: $ui-background;
+
+					&:not(.active) {
+						stroke: $text-02;
+						fill: $ui-background;
+					}
+				}
+			}
+		}
+	}
+}
+
+$gray100: #171717;
```

Comment:
You know what I'm going to say 🙂

Reply:
this looks nice to map the colors to tokens 

```
.carbon--theme--g90 {
	@include carbon--theme($carbon--theme--g90) {
		$ui-01: $carbon--gray-100;
		g.cc-scatter {
			circle.dot.unfilled {
				fill: $ui-01;
			}
		}
	}
}
```


but if you look at core/src/styles/components/_grid.scss, the tokens dont map across the themes that well, so it wouldnt make sense to re-map a bunch of them for the purpose of semantics when it would decrease the readibility


```
.carbon--theme--g10 {
	@include carbon--theme($carbon--theme--g10) {
		.cc-grid {
			rect.chart-grid-backdrop {
				fill: $ui-01;
			}

			g.x.grid,
			g.y.grid {
				g.tick line {
					stroke: $ui-03;
				}

				g.tick.active line {
					stroke: $focus;
				}
			}
		}
	}
}

.carbon--theme--g90 {
	@include carbon--theme($carbon--theme--g90) {
		.cc-grid {
			rect.chart-grid-backdrop {
				fill: $carbon--gray-100;
			}

			g.x.grid,
			g.y.grid {
				g.tick line {
					stroke: $ui-01;
				}

				g.tick.active line {
					stroke: $focus;
				}
			}
		}
	}
}
```

so im going to consistently update to using color mapping -> $carbon--gray-100 in the case where the colors arent tokenized properly

Reply:
Cool that makes sense 👍

Reply:
Also, is gray 100 not available in `@carbon/colors`? If not, then we should open up a PR